src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,15 @@
+package core.basesyntax;
+import java.util.Random;
+
+public class Lottery {
+
+  Random random = new Random();
+  ColorSupplier colorSupplier = new ColorSupplier();
```

Comment:
Consider making 'random' and 'colorSupplier' final since they are not intended to be changed after initialization.